examples/examples-release-17/pom.xml

```diff
@@ -107,7 +107,6 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-deploy-plugin</artifactId>
-				<version>3.1.1</version>
```

Comment:
when the version is set in dependencyManagement section of the parent, you don't need to repeat it in any child module.